src/diff/index.js

```diff
@@ -331,7 +331,7 @@ export function unmount(vnode, ancestorComponent, skipRemove) {
 
 	let dom;
 	if (!skipRemove && vnode._lastDomChild==null) {
```

Comment:
If I understand the description correctly, could we just change this if statement to something like:
```js
if (!skipRemove && typeof vnode.type !== 'function') {
```
The `typeof ... 'function'` check should catch components and Fragments and only enter this block for text vnodes (`vnode.type == null`) and DOM vnode (`typeof vnode == 'string'`)

Reply:
Hmm, I'll test it. It could be a better solution yes. This however works further on the other branch and I heard you were working on something as well. So we might have to combine our branches or?